javalanglib/src/main/scala/java/lang/Character.scala

```diff
@@ -254,6 +254,32 @@ object Character {
   @inline def toCodePoint(high: scala.Char, low: scala.Char): Int =
     ((high & SurrogateUsefulPartMask) << 10) + (low & SurrogateUsefulPartMask) + 0x10000
 
+
+  def codePointCount(seq: CharSequence, beginIndex: Int, endIndex: Int): Int = {
```

Comment:
We already have an implementation of this functionality in `java.lang.String`:
https://github.com/scala-js/scala-js/blob/bb4835763b9316c8370bc00b6bd4f6c0653a0e95/javalanglib/src/main/scala/java/lang/_String.scala#L82-L94
Since a String is a CharSequence, we should move the existing implementation of `java.lang.String.codePointCount` here, and change the former to forward to this one.

But clearly we should not invent a new implementation.